src/promptflow-tools/promptflow/tools/common.py

```diff
@@ -599,3 +595,7 @@ def init_azure_openai_client(connection: AzureOpenAIConnection):
 
     conn_dict = normalize_connection_config(connection)
     return AzureOpenAIClient(**conn_dict)
+
+
+def is_on_runtime() -> bool:
```

Comment:
`in_local_env`? And please add a comment because the current condition is not that accurate.

Reply:
agree. The word "runtime" is really confused

Reply:
updated